configure.ac

```diff
@@ -1669,6 +1669,9 @@ else
     fi
     AC_MSG_RESULT($use_natpmp_default)
     AC_DEFINE_UNQUOTED([USE_NATPMP], [$natpmp_setting], [NAT-PMP support not compiled if undefined, otherwise value (0 or 1) determines default state])
+    if test x$TARGET_OS = xwindows; then
+      NATPMP_CPPFLAGS="-DSTATICLIB -DNATPMP_STATICLIB"
```

Comment:
> `STATICLIB` is also included, because both of these defines used to be `STATICLIB`, so if someone wass using an older source (actually the commit after what is in our current tarball, [miniupnp/libnatpmp@d5d2d6d](https://github.com/miniupnp/libnatpmp/commit/d5d2d6d74a25f019c367c4a1942e98906b97cb89)) , they would need that defined instead.

I'm not follow why do we need `-DSTATICLIB` now?